drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -54,9 +57,15 @@ def eval_rule(rule)
       when 'EQUAL'
         # { variable: 'projectType', operator: 'EQUAL', value: 1 }
         eval_var(rule.fetch('variable')) == rule.fetch('value')
+      when 'NOT_EQUAL'
+        # { variable: 'projectType', operator: 'NOT_EQUAL', value: 1 }
+        eval_var(rule.fetch('variable')) != rule.fetch('value')
       when 'INCLUDE'
         # { variable: 'projectFunders', operator: 'INCLUDE', value: 1 }
         eval_var_multi(rule.fetch('variable')).include?(rule.fetch('value'))
+      when 'NOT_INCLUDE'
+        # { variable: 'projectFunders', operator: 'NOT_INCLUDE', value: 1 }
+        !eval_var_multi(rule.fetch('variable')).include?(rule.fetch('value'))
```

Comment:
maybe?
https://apidock.com/rails/Enumerable/exclude%3F
```suggestion
        eval_var_multi(rule.fetch('variable')).exclude?(rule.fetch('value'))
```